static/app/views/alerts/wizard/index.tsx

```diff
@@ -89,7 +97,16 @@ class AlertWizard extends React.Component<Props, State> {
         renderDisabled={renderNoAccess}
       >
         {({hasFeature}) => (
-          <WizardButtonContainer>
+          <WizardButtonContainer
```

Comment:
not sure if this is too weird to put the onclick handler on the button container rather than the create alert button